File: Android/divyaagarwal24_DivyaAgarwal_2024it1100_2/Patterns/pattern3.java
import java.util.*;

public class pattern3 {

    public static void main(String[] args) {
        Scanner scn = new Scanner(System.in);
        int i,j,n;
        n=scn.nextInt();
        for(i=0;i<n;i++)
        {
            for(j=n;j>(i+1);j--)
            {
                System.out.print("\t");
            }
            for(j=0;j<=i;j++)
            System.out.print("*\t");
            System.out.println();

        }// write ur code here

    }
}

File: Android/AnjumanHasan_AnjumanHasan_2024cse1157_2/Patterns/pattern3.cpp
#include <iostream>
using namespace std;

int main(){
    int n,i,j;
    cin >> n;
    
   //write your code here
   char A[n][n];
   for (int i = 0; i <=n; i++)
   {
        for (int j = 0; j <= n; j++)
      {
         if(i+j<n-1)
         {
             A[i][j]=' ' ;
         }
         else
         A[i][j]='*';
      }
      
   }
    for (int i = 0; i < n; i++)
   {
        for (int j = 0; j < n; j++)
      {
          cout <<  A[i][j] << "\t" ;
      }
      cout << endl;
   }
   return 0;
}
